metricbeat/module/system/core/config.go

```diff
@@ -0,0 +1,46 @@
+package core
+
+import (
+	"strings"
+
+	"github.com/elastic/beats/libbeat/logp"
+	"github.com/pkg/errors"
+)
+
+// Core metric types.
+const (
+	percentages = "percentages"
+	ticks       = "ticks"
+)
+
+// Config for the system core metricset.
+type Config struct {
+	Metrics  []string `config:"core.metrics"`
+	CPUTicks *bool    `config:"cpu_ticks"` // Deprecated.
```

Comment:
I don't like deprecated features in the code base. If don't do it now, we have to wait until 7.0 to remove it. Seems like a good time to do it now. Any disadvantage doing it now?